src/AppBundle/Form/BaptismSearchType.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace AppBundle\Form;
+
+//use AppBundle\Entity\Event;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class BaptismSearchType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add(
+            'city',
+            TextType::class, array(   // set before entity exist
```

Comment:
Please remove unnecessary comments, it should be one line : 
$builder->add('city', TextType::class) 
since you're not adding anything, the array is unnecessary.